src/client/cloud.rs

```diff
@@ -84,12 +84,16 @@ impl CapellaClient {
         mac.update(bearer_payload.as_bytes());
         let mac_result = mac.finalize();
 
-        let bearer = format!(
+        let mut bearer = format!(
```

Comment:
Rather than changing this to mut, and then generating a token, just to overwrite it, why not just generate the appropriate kind of bearer token for the API that's in use directly?